src/css/BlogList.module.css

```diff
@@ -44,18 +36,17 @@
 }
 
 .snap {
-    position: absolute;
-    width: 570px;
-    height: 342px;
-    left: 76px;
-    top: 168px;
-
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
 }
 
 @media screen and (max-width: 840px) {
     .Post {
```

Comment:
In the future, we need more responsive layout--many of users see this on mobile.

Reply:
I see, but we have reviewed the design on mobile and they dont break on mobile. We're following the same pattern that the initial project had, using  -> @media screen and (max-width: 840px) to specify layout specific to device with mobile screen sizes.